CRM/Contribute/Form/Task/Batch.php

```diff
@@ -215,6 +215,7 @@ public function postProcess() {
         if ('Completed' === $currentStatus &&
           in_array($contribution['contribution_status_id:name'], ['Pending', 'Partially paid'], TRUE)) {
           Payment::create(FALSE)
+            ->setnotificationForCompleteOrder(FALSE)
```

Comment:
It looks like a typo, though `Civi\Api4\Action\Payment\Create` has the same typo in its docblock...

Reply:
Good spot @totten - fixed